README.md

```diff
@@ -195,8 +195,7 @@ const { flags, values, positionals } = parseArgs(argv, options);
 - Does the API specify whether a `--` was present/relevant?
   - no
 - Is `-foo` the same as `--foo`?
-  - yes <-- ! kind of a blocker for shortopts !
-  - Recommend: "No, -foo is shortopts form of --f --o --o" (assuming none are defined, or withValues)
+  - no, `-foo` is a short option or options (WIP: https://github.com/pkgjs/parseargs/issues/2)
```

Comment:
iow, `-foo` must be either `-f -o -o` or `-f oo`?

Reply:
Or `-f -o o`.

Reply:
oof, that sounds like a horrific result, because how would i know at a glance what it means if that were a possibility?

Reply:
This is not a common or encouraged usage. But it is a possibility.

In this scenario the person who wrote the command invocation knows that `-o` takes a value. They know that you may put the value directly after the short option. They know that you may combine short options. (And if they did this in a script then apparently they don't care about the readability and maintainability!)

It might look more reasonable with a progression, and one feature at a time:
```
start --verbose --port 80
start -v -p 80
start -v -p80
start -vp80
```

Real world example:
```
% tar -xff
tar: Error opening archive: Failed to open 'f'
```

Reply:
That combination of obscure knowledge doesn't seem worth it to me to be able to do `-xff` instead of being forced to add a space and a dash (`-x -ff`).

Reply:
And as a CLI user, you never need to know it is even a possibility! Which is how it should be. It should be easy to use the CLI.

Reply:
I totally agree - which is why I think the only two options should be `-f -o -o` (my preference) or `-f oo`, but never `-f -o o` or `-f o -o`.

Reply:
Well, we agree it won't be `-f o -o`. 😄

Reply:
How is `-f o -o` any different than `-f -o o`? The order shouldn't matter for this scenario, only if it's declared withValue.

Reply:
Perhaps we should make the example is `-bar` the same as `--bar`, so there isn't confusion due to the `oo`.

I think in both cases it should be: `-b -a -r`, and `-f -o -o`, and the multiple `o`s would just clobber each other.

We could eventually consider adding support for multiples so that `-o`, `-o` would be `o: 2`, but I think this also supports the `-f`, `-o`, `-o`, interpretation.

Reply:
Less issues with `--bar` but the key point is the answer is now "no". That is a step forward for the README!

Reply:
@bcoe i'm 100% fine with what you think it should be in that comment. I found "`-bar` could be `-b a -r` or `-b -a r`, no matter how they're configured, both to be unacceptable.

Reply:
I'll do some fresh exploration on potential `-bar` equivalence to `-b -a r`. I went deep on this a couple of years ago because of bugs reported against Commander which at the time did a blind expansion of `-foo` to`-f -o -o`, and separately requests and PR for support for `-p80`. So I have been here before.